src/irr/asset/COBJMeshFileLoader.cpp

```diff
@@ -285,7 +285,7 @@ asset::SAssetBundle COBJMeshFileLoader::loadAsset(io::IReadFile* _file, const as
 					core::vectorSIMDf simdNormal;
 					simdNormal.set(normalsBuffer[Idx[2]].data);
                     simdNormal.makeSafe3D();
-					v.normal32bit = _params.quantNormalCache.quantizeNormal2_10_10_10(simdNormal);
+					v.normal32bit = _params.quantNormalCache.quantizeNormal<E_QUANT_NORM_CACHE_TYPE::Q_2_10_10_10>(simdNormal);
```

Comment:
remember about the fallback if `_params.quantNormalCache==nullptr` (get the default from assetManager->getMeshManipulator()->getNormalCache())